### PR TITLE
Use context managers to simplify log serve management

### PR DESCRIPTION
**Pull request by dstandish**
It's a little cleaner.  That is all.

### CODE REVIEW

File: airflow/cli/commands/celery_command.py [hunk 1]
```diff
@@ -197,10 +192,13 @@ def worker(args):
                 stderr=stderr_handle,
             )
             with ctx:
-                _run_worker(options=options, skip_serve_logs=skip_serve_logs)
+                with _serve_logs(skip_serve_logs):
```
Comment (potiuk) [inline airflow/cli/commands/celery_command.py]: `with ctx, _serve_logs(skip_serve_logs):` ?

**Review by potiuk — APPROVED** [review]
Nice one. One really small nit :)

File: airflow/cli/commands/celery_command.py (hunk 1 above)
Comment (dstandish) [reply airflow/cli/commands/celery_command.py]: let me improve the naming of `ctx`

**Comment by dstandish** [comment]
done

**Pull request merged**